src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -860,7 +871,13 @@ public static function get_resolved_theme_uris( $theme_json ) {
 			return $resolved_theme_uris;
 		}
 
-		$theme_json_data = $theme_json->get_raw_data();
+		$theme_json_data               = $theme_json->get_raw_data();
+		$resolved_theme_uris_cache_key = md5( wp_json_encode( $theme_json_data ) );
```

Comment:
Might be premature. 

Looks like there's a small impact on performance on TT4

https://github.com/WordPress/wordpress-develop/actions/runs/11432958553?pr=7596

I'll run again as the tests vary between commits, e.g., https://github.com/WordPress/wordpress-develop/actions/runs/11432437736?pr=7596